src/Microsoft.DotNet.Cli/Program.cs

```diff
@@ -29,6 +32,13 @@ public class Program
   run           Compiles and immediately executes a .NET project
   repl          Launch an interactive session (read, eval, print, loop)
   pack          Creates a NuGet package";
+        private static readonly string ProductVersion = GetProductVersion();
```

Comment:
What do you mean? Expose the version of `dotnet.exe` or just expose the version of each command executable? For now I'm going to leave this in order to unblock the PR, I really just added it to have a place to display the RID :)